infrastructure/cdn-in-a-box/traffic_portal_integration_test/Dockerfile

```diff
@@ -48,6 +48,7 @@ RUN if [[ "${RHEL_VERSION%%.*}" -eq 7 ]]; then \
 FROM os-dependencies AS node-dependencies
 # Download and install node
 RUN curl -sL https://rpm.nodesource.com/setup_18.x | bash -
+RUN dnf install -y nodejs
```

Comment:
This should use yum rather than dnf for Centos7 backward compatibility

Reply:
There is a line at the beginning to ensure `dnf` is installed as the rest of the Dockerfile depends on it as well.

Reply:
I take it back, we did the bits to add dnf backward into Centos7 and dnf is consistent with the rest of the dockerfile